easybuild/easyconfigs/n/NanoPlot/NanoPlot-1.28.4_fix_deps.patch

```diff
@@ -0,0 +1,13 @@
+fix hardcoded statsmodel version
+author: Pavel Grochal (INUITS)
+--- NanoPlot-1.28.4/setup.py.orig	2020-02-18 08:28:53.000000000 +0100
++++ NanoPlot-1.28.4/setup.py	2020-03-22 17:34:05.929501866 +0100
+@@ -41,7 +41,7 @@
+                       'nanoget>=1.9.0',
+                       'nanomath>=0.23.1',
+                       "pauvre==0.1.86",
+-                      "statsmodels==0.10.1",
++                      "statsmodels>=0.10.1",
```

Comment:
I think we can avoid using a patch file here (since that implies we need to maintain it).

Instead, we can use `sed` in `preinstallopts` (since we don't have "native" support for runtime patching yet in EasyBuild):

```python
preinstallopts = "sed -i'' 's/==/>=/g' setup.py && "
```

This results in these changes:

```diff
$ diff -u setup.py.orig setup.py
--- setup.py.orig       2020-03-23 10:00:34.275778000 +0100
+++ setup.py    2020-03-23 10:00:41.706679279 +0100
@@ -40,8 +40,8 @@
                       'matplotlib>=2.1.0',
                       'nanoget>=1.9.0',
                       'nanomath>=0.23.1',
-                      "pauvre==0.1.86",
-                      "statsmodels==0.10.1",
+                      "pauvre>=0.1.86",
+                      "statsmodels>=0.10.1",
                       'plotly>=4.1.0',
                       ],
     package_data={'NanoPlot': []},

Reply:
unfortunately `pauvre==0.1.86` needs to stick to this version. But will change the it to the sed :) instead of patchfile.

Reply:
Can you clarify why sticking to `pauvre==0.1.86` is required? Are there breaking changes in the `0.19xx` versions?

Reply:
I'm sorry my bad with pip check, I've bumped version to latest.